src/main/io/osd.c

```diff
@@ -667,9 +707,15 @@ void osdUpdateAlarms(void)
         CLR_BLINK(OSD_FLYTIME);
 
     if (getMAhDrawn() >= osdConfig()->cap_alarm)
+    {
```

Comment:
Put the opening bracket on the same line as the `if` statement. (New line is only used for function definitions.)